zebra/rib.h

```diff
@@ -166,6 +166,11 @@ struct route_entry {
 	struct opaque *opaque;
 };
 
+#define RKERNEL_ROUTE(type) ((type) == ZEBRA_ROUTE_KERNEL)
```

Comment:
we've got a copy of these in `rt.h` too - it'd be better to have just one version in zebra?

Reply:
meant to remove them in rt.h